receiver/mongodbatlasreceiver/README.md

```diff
@@ -36,8 +36,10 @@ MongoDB Atlas [Documentation](https://www.mongodb.com/docs/atlas/reference/api/l
   - `secret` (required if using `listen` mode)
   - `endpoint` (required if using `listen` mode)
   - `poll_interval` (default `5m`, only relevant using `poll` mode)
-  - `max_alerts` (default `1000`)
-    - When in `poll` mode, this is the maximum number of alerts the receiver will attempt to process per poll. Alerts are processed from most recently created to least recently created.
+  - `page_size` (default `100`)
+    - When in `poll` mode, this is the number of alerts that will be processed per request to the MongoDB Atlas API per project.
```

Comment:
```suggestion
    - When in `poll` mode, this is the number of alerts that will be processed per request to the MongoDB Atlas API.
```
This isn't really a variable in the context of projects. It's just going to always be the page size we request.